Odata-docs/client/v6/batch-operations.md

```diff
@@ -164,3 +164,61 @@ The request Payload is as following:
 	--changeset_b98a784d-af07-4723-9d5c-4722801f4c4d--
 	--batch_06d8a02a-854a-4a21-8e5c-f737bbd2dea8--
 ```
+
+## Relative Uris in Batch Requests
+`ODataServiceContext` provides `SaveChangesOptions.UseRelativeUri` which allows individual requests in batch to use Relative Uris. In this case, they will rely on the Base Uri of the top level request.
+
+### Relative Uris in Batch Queries
+We pass `SaveChangesOptions.BatchWithSingleChangeset` or `SaveChangesOptions.BatchWithIndependentOperations` with `SaveChangesOptions.UseRelativeUri` in the ExecuteBatch and ExecuteBatchAsync Methods af follows.
+
+```
+var batchResponse = dsc.ExecuteBatch(SaveChangesOptions.BatchWithIndependentOperations | SaveChangesOptions.UseRelativeUri, peopleQuery, airlinesQuery);
+```
+```
+var batchResponse = await dsc.ExecuteBatchAsync(SaveChangesOptions.BatchWithIndependentOperations | SaveChangesOptions.UseRelativeUri, peopleQuery, airlinesQuery);
```

Comment:
It dont think it's necessary to show both examples. Just one should suffice with an indication that the same applies for the async versions. Maybe even a quick example of the kind of error it helps fix. So a scenario/example with before (with error[s]) and after relative URI

Reply:
I don't think showing errors that were there before this feature is prudent. My assumption is we are documenting a new feature not a fix to some issue.